workspace/assets/css/dev/dev.less

```diff
@@ -10,6 +10,18 @@
 	outline: rgba(255,0,0,36%) 1px solid;
 	outline-offset: 8px;
 }
+
+.show-img-without-alt {
+	img {
+		border: 2px solid red;
+	}
```

Comment:
Should we ignore images without alt inside an area-hidden/role=presentation?

I mean, technically, the screen reader will skip the content, but on the other hand, Google might expect an alt 🤷‍♂️ .

Reply:
its not for the screen reader. its only for the dev tools. will never be in prod...

Reply:
Yeah I know, but the fact that an image that has no alt inside a aria-hidden should not be an issue and called for, imo.

Reply:
@fhamon is right, but I would keep the warning anyhow.

Reply:
Yeah.. Its only to found potential error.

Reply:
👍